### PR TITLE
feat: publishing schema

### PR DESCRIPTION
**Pull request by yehjxraymond**
## Goal
This PR aims to publish the schema at a fixed url. This allows us to set the JSON schema `$id` correctly instead of simply setting to a random string. Setting it to a random string will cause the schema to fail validation, resulting in many online validators rejecting the JSON schema.

## Solution
Adds a new script to create a public folder and move the schema files in. 
This is meant to be used by netlify to publish the two schema. 

The intended url is as followed:

- https://schema.openattestation.com/2.0/schema.json
- https://schema.openattestation.com/3.0/schema.json

On netlify the build command will be `npm run publish:schema` and folder will be `public`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
     "test:watch": "jest --watch",
     "lint": "eslint . --ext .ts --max-warnings 0",
     "lint:fix": "npm run lint -- --fix",
-    "postinstall": "./scripts/postinstall.sh"
+    "postinstall": "./scripts/postinstall.sh",
```
Comment (Nebulis) [inline package.json]: Hmmmmm I dont understand the diff ? i've changed this to use node script to fix #70 (https://github.com/Open-Attestation/open-attestation/blob/master/package.json#L23)

Comment (Nebulis) [reply package.json]: (ok the MR is in conflict :D)

Comment (yehjxraymond) [reply package.json]: lol fixed.

**Pull request merged**